src/ui/units/workbooks/components/WorkbookPage/WorkbookPage.tsx

```diff
@@ -38,6 +41,8 @@ import {WorkbookTabs} from '../WorkbookTabs/WorkbookTabs';
 import {TAB_ALL} from '../WorkbookTabs/constants';
 import {TabId} from '../WorkbookTabs/types';
 
+import PencilToLineIcon from '@gravity-ui/icons/svgs/pencil-to-line.svg';
```

Comment:
Let's use import from the lib root – `import {PencilToLine} from '@gravity-ui/icons';`

Reply:
fixed